### PR TITLE
SCA: minor code tweaks

### PR DESCRIPTION
**Pull request by kalessil**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

- minor code tweaks
- drop private properties, which used as local variables

### CODE REVIEW

**Review by chalasr — APPROVED** [review]
Thanks!

File: src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php [hunk 1]
```diff
@@ -104,12 +104,11 @@ public function configure(Event $event = null)
             if ($event instanceof KernelEvent) {
                 if (method_exists($kernel = $event->getKernel(), 'terminateWithException')) {
                     $request = $event->getRequest();
-                    $hasRun = &$this->hasTerminatedWithException;
-                    $this->exceptionHandler = function (\Exception $e) use ($kernel, $request, &$hasRun) {
-                        if ($hasRun) {
+                    $this->exceptionHandler = function (\Exception $e) use ($kernel, $request) {
```
Comment (nicolas-grekas) [inline src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php]: Another option here is to make the closure `static`. Thos would have the benefit of removing a circular reference.

File: src/Symfony/Bundle/FrameworkBundle/Tests/Command/RouterMatchCommandTest.php [hunk 2]
```diff
@@ -78,11 +78,7 @@ private function getKernel()
             ->expects($this->atLeastOnce())
             ->method('has')
             ->will($this->returnCallback(function ($id) {
-                if ('console.command_loader' === $id) {
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/Tests/Command/RouterMatchCommandTest.php]: this applies to 3.4 also - maybe others too? they should be submitted against 3.4

File: src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php (hunk 1 above)
Comment (kalessil) [reply src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php]: I like the proposal, would you consider enabling `StaticLambdaFixer` CS fixer rule (I have to revert this specific change then)?

File: src/Symfony/Bundle/FrameworkBundle/Tests/Command/RouterMatchCommandTest.php (hunk 2 above)
Comment (kalessil) [reply src/Symfony/Bundle/FrameworkBundle/Tests/Command/RouterMatchCommandTest.php]: I'll re-check the PR changes against 3.4, thanks for reminding =)

File: src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php (hunk 1 above)
Comment (nicolas-grekas) [reply src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php]: please revert and add `static` instead yes
dunno about `StaticLambdaFixer` - why not :)

Comment (kalessil) [reply src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php]: Will do + will create a PR for static closures.

Comment (kalessil) [reply src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php]: BTW which branch to target with static closures, 3.4?

Comment (nicolas-grekas) [reply src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php]: > BTW which branch to target with static closures, 3.4?
maybe none: the rule is risky and I'm not sure it makes sense to apply it globally now that I reviewed what it does :)

Comment (kalessil) [reply src/Symfony/Component/HttpKernel/EventListener/DebugHandlersListener.php]: Got it, but at least you are now aware of it =)

**Comment by nicolas-grekas** [comment]
Thank you @kalessil.

File: src/Symfony/Component/Cache/Adapter/SimpleCacheAdapter.php [hunk 3]
```diff
@@ -13,13 +13,12 @@
...
 use Psr\SimpleCache\CacheInterface;
 use Symfony\Component\Cache\PruneableInterface;
-use Symfony\Component\Cache\ResettableInterface;
 use Symfony\Component\Cache\Traits\ProxyTrait;
 
 /**
  * @author Nicolas Grekas <p@tchwork.com>
  */
-class SimpleCacheAdapter extends AbstractAdapter implements PruneableInterface, ResettableInterface
+class SimpleCacheAdapter extends AbstractAdapter implements PruneableInterface
```
Comment (kunicmarko20) [inline src/Symfony/Component/Cache/Adapter/SimpleCacheAdapter.php]: Isn't this a BC break?

File: src/Symfony/Component/Workflow/Exception/InvalidTokenConfigurationException.php [hunk 4]
```diff
@@ -16,6 +16,6 @@
  *
  * @author Matt Johnson <matj1985@gmail.com>
  */
-class InvalidTokenConfigurationException extends LogicException implements ExceptionInterface
+class InvalidTokenConfigurationException extends LogicException
```
Comment (kunicmarko20) [inline src/Symfony/Component/Workflow/Exception/InvalidTokenConfigurationException.php]: Same here?

File: src/Symfony/Component/Cache/Adapter/SimpleCacheAdapter.php (hunk 3 above)
Comment (kunicmarko20) [reply src/Symfony/Component/Cache/Adapter/SimpleCacheAdapter.php]: Ignore me, base class implements it

**Pull request merged**